src/peep/peep.c

```diff
@@ -10614,4 +10614,7 @@ void game_command_set_peep_name(int *eax, int *ebx, int *ecx, int *edx, int *esi
 		(uint8*)ebp,
 		(uint8*)edi
 		);
+	}
```

Comment:
> Without the indent there, the code doesn't work

Indentation should never change if the code works or not. This is C, not python.